.github/workflows/main.yml

```diff
@@ -46,3 +46,26 @@ jobs:
         working-directory: git-sync
         run: |
           make test
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20.6'
+
+      - uses: actions/checkout@v3
+        with:
+          path: git-sync
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
```

Comment:
Should we copy the k/k .golangci-lint config?

Reply:
Sure, I can pull it in :-)  Might as well have a consistent style.